src/main/java/io/reactivex/internal/functions/ObjectHelper.java

```diff
@@ -121,6 +121,20 @@ public static long verifyPositive(long value, String paramName) {
         }
         return value;
     }
+    
+    /**
+     * Validate that the given index is greater than equal to zero or report
+     * an IndexOutOfBoundsException with the count.
+     * @param count the value to validate
+     * @return count
+     * @throws IndexOutOfBoundsException if count &lt; 0
+     */
+    public static long verifyIndexPositive(long count) {
```

Comment:
This should be `verifyCountPositive` if the message and argument are named `count`.